config/samples/example-cruisecontroloperation.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kafka.banzaicloud.io/v1alpha1
+kind: CruiseControlOperation
+metadata:
+  name: example-cruisecontroloperation
+  namespace: kafka
+spec:
+  errorPolicy: retry
+  
```

Comment:
**Note**: this looks weird that GitHub states no new line at the end of the file, while line 8 is supposed to be that.
I think your editor added spaces to the new line when you hit return and that is the reason.
I can let this go, because the main reason is to have a blank line if you are reading from a terminal emulator which is fulfilled.